modules/raft/src/main/java/org/apache/ignite/raft/jraft/Node.java

```diff
@@ -317,4 +317,14 @@ public interface Node extends Lifecycle<NodeOptions>, Describer {
      * @return node's current term.
      */
     long getCurrentTerm();
+
+    /**
+     * Returns {@code true} if node is currently in the process of installing a snapshot.
+     */
+    boolean isInstallingSnapshot();
+
+    /**
+     * Returns the value of last replicated log index. Corresponding log entry might not yet be written to the log storage (no flush).
+     */
+    long lastLogIndex();
```

Comment:
Maybe rename to `lastReplicatedLogIndex` ?

Reply:
No, everywhere else it's called last log index, I don't want to create confusion with alternative naming.